packages/api/src/models/medical/cq-directory.ts

```diff
@@ -0,0 +1,59 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { CQOrganization } from "../../domain/medical/cq-directory";
+import { BaseModel, ModelSetup } from "../../models/_default";
+import { Organization } from "@metriport/carequality-sdk/models/organization";
+
+export class CQDirectoryModel extends BaseModel<CQDirectoryModel> implements CQOrganization {
+  static NAME = "cq_directory";
```

Comment:
ditto: `cq_directory_entry`?

Reply:
for the table name, i think it makes sense to call it cq directory - this is essentially what it is, just a local copy of it

Reply:
you don't name tables based on what they are, you name tables based on what their rows represent

ie it's not `customer_list`, it's `customer` - so this is `cq_directory_entry`

besides, my comment about changing this model's name to `CQOrganizationEntry` was not addressed?